tests/unit/serialization/compact_test.py

```diff
@@ -80,7 +80,7 @@ def test_schema(self):
 
 class SchemaTest(unittest.TestCase):
     def test_constructor(self):
-        fields = [FieldDescriptor(kind.name, kind) for kind in FieldKind]
+        fields = [FieldDescriptor(kind.name, kind) for kind in FieldKind if kind is not None]
```

Comment:
`kind is not None` is true always isn't it?

Reply:
Changed at: https://github.com/hazelcast/hazelcast-python-client/pull/550/commits/0243e0c4c9c8622a2cd46c526cbe4d705638efd6